lib/board.rb

```diff
@@ -9,50 +9,31 @@ def initialize
     }
   end
 
-  def cell_empty?(selected_cell)
-    return true if @cells[selected_cell.to_sym] == ' '
+  def cell_empty?(cell)
+    return true if @cells[cell] == ' '
```

Comment:
Looks like blank? is a rails-only method, but empty? worked like a charm! Thanks!